### PR TITLE
Increase PowerMax Code Coverage

### PR DESCRIPTION
**Pull request by doriac11**
# Description
- Fix broken Acceptance tests.
- Increase the code coverage.
- Bug fixes.

# ISSUE TYPE
- Bugfix Pull Request

##### RESOURCE OR DATASOURCE NAME
All

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit tests
- [x] Acceptance tests

image

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

**Comment by doriac11** [comment]
run e2e test

File: powermax/helper/maskingview_helper.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	createMaskingViewParam.SetPortGroupSelection(portGroupSelection)
+	createMaskingViewParam.SetStorageGroupSelection(storageGroupSelection)
+
+	maskingViewReq := client.PmaxOpenapiClient.SLOProvisioningApi.CreateMaskingView(ctx, client.SymmetrixID)
+	maskingViewReq = maskingViewReq.CreateMaskingViewParam(*createMaskingViewParam)
+	return maskingViewReq.Execute()
+}
+
+// CreateMaskingView Creates a new masking view
+func GetMaskingView(ctx context.Context, client client.Client, name string) (*pmax.MaskingView, *http.Response, error) {
```
Comment (doriac11) [inline powermax/helper/maskingview_helper.go:55]: Made change like these throughout the PR, this makes it easier to mock the REST requests in the tests

**Comment by doriac11** [comment]
run e2e tests

**Comment by doriac11** [comment]
run e2e test

**Comment by doriac11** [comment]
run e2e test

**Review by nikitajoshi1 — DISMISSED** [review]

**Review by shekhar-j — DISMISSED** [review]

**Comment by doriac11** [comment]
run e2e test

**Review by nikitajoshi1 — DISMISSED** [review]

**Comment by doriac11** [comment]
run e2e test

**Review by shekhar-j — APPROVED** [review]

**Pull request merged**